utils/cli/core/core.py

```diff
@@ -97,6 +98,16 @@ def tasks_create(self, name, labels, overlap, segment_size, bug, resource_type,
                 log.info(logger_string)
 
             self.tasks_upload(task_id, annotation_format, annotation_path, **kwargs)
+        if dataset_repository_url:
+            git_url = self.api.git
+            response = self.session.post(
+                        f'{git_url}create/{task_id}',
+                        json={
+                            'path': dataset_repository_url,
+                            'lfs': lfs,
+                            'tid': task_id})
+            response_json = response.json()
+            log.info(f"Create: {response_json.rq_id}")
```

Comment:
Please check the response `status_code` here, the request may fail.
Also, seems the line should be changed `response_json.rq_id -> response_json['rq_id']`

Additional comment: I think it is necessary to check the status of add of the remote repo with `{git_url}/check_process` to make sure everything is ok, or the print error message

Reply:
Updated. Please confirm that this is satisfactory.